### PR TITLE
replace keras example with a link. fix #599

### LINKED ISSUE #599: Pytorch interface should be emphasized in README, Keras interface should be moved to a different file with proper disclaimers
As pytorch is the best-supported python frontend, it should be placed first in the readme and the keras interface (which is much less supported) should probably be moved to a separate file linked to by the readme instead.

### PR DESCRIPTION
**Pull request by xinhaoc**
**Description of changes:**

**Related Issues:**

Linked Issues:
- Issue #599 

Issues closed by this PR:
- Closes #599 

**Before merging:**

- [ ] Did you update the [flexflow-third-party](https://github.com/flexflow/flexflow-third-party) repo, if modifying any of the Cmake files, the build configs, or the submodules?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,10 +33,13 @@ def top_level_task():
 
 **More FlexFlow PyTorch examples**: see the [pytorch examples folder](https://github.com/flexflow/FlexFlow/tree/master/examples/python/pytorch).
 
+## TensorFlow Keras Support
+Users can use FlexFlow to accelerate the training procedure of existing TensorFlow Keras models, please read the [Keras Support](https://flexflow.readthedocs.io/en/latest/keras.html).
```
Comment (lockshaw) [inline README.md:37]:
```suggestion
FlexFlow prioritizes PyTorch compatibility, but also includes frontends for [Tensorflow Keras](https://flexflow.readthedocs.io/en/latest/keras.html) and [ONNX](...) models.
```

Comment (lockshaw) [reply README.md:37]: Also, @gabrieleoliaro correct me if I'm wrong here but shouldn't this be a local md link that RTD converts into a full url?

**Review by lockshaw — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gabrieleoliaro) [reply README.md:37]: yes, basically, if you put the link to the corresponding `.md` file here, RTD will automatically convert that into an URL when building the docs :)

Comment (xinhaoc) [reply README.md:37]: the link should be a local file?

Comment (lockshaw) [reply README.md:37]: Yes, it should link to the corresponding md file in this repo (the RTD is automatically generated from the contents of this repo)

File: README.md [hunk 2]
```diff
@@ -64,10 +33,13 @@ def top_level_task():
 
 **More FlexFlow PyTorch examples**: see the [pytorch examples folder](https://github.com/flexflow/FlexFlow/tree/master/examples/python/pytorch).
 
+## TensorFlow Keras Support
+FlexFlow prioritizes PyTorch compatibility, but also includes frontends for [Tensorflow Keras](./docs/source/keras.rst) and [ONNX](...) models.
```
Comment (gabrieleoliaro) [inline README.md:37]: looks good! Can we just add the onnx link here as well? should be `./docs/source/onnx.rst`

**Review by gabrieleoliaro — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (xinhaoc) [reply README.md]: fixed

File: README.md [hunk 3]
```diff
@@ -64,10 +33,13 @@ def top_level_task():
 
 **More FlexFlow PyTorch examples**: see the [pytorch examples folder](https://github.com/flexflow/FlexFlow/tree/master/examples/python/pytorch).
 
+## TensorFlow Keras Support
```
Comment (lockshaw) [inline README.md:36]: Let's merge this with the next sections. It's a bit odd to say that we have ONNX support twice :slightly_smiling_face:

**Review by lockshaw — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (xinhaoc) [reply README.md:36]: got it

**Review by gabrieleoliaro — APPROVED** [review]

**Review by lockshaw — APPROVED** [review]

**Pull request merged**